migration.rb

```diff
@@ -0,0 +1,59 @@
+class MigrateWf
+  attr_accessor :failed_account_ids, :req_counter, :start_time, :options
+
+  def initialize
+    @failed_account_ids = []
+    @req_counter = 0
+    @start_time = Time.now.to_i
+    @options = { payload_type: 'workflow_sync', meta: { 'source' => 'search' } }
+  end
+
+
+  def check_redis
+    BaseRedis.key_exists?("STOP_WF_SEARCH_MIGRAION")
```

Comment:
why do we ened this?

Reply:
to stop scripts execution. 
`break if check_redis` we're breaking the loop and stopping the execution